spreadsheet/formula/evaluator.go

```diff
@@ -21,12 +26,62 @@ func NewEvaluator() Evaluator {
 }
 
 type defEval struct {
+	isRef bool
+	booleans []bool
 }
 
 func (d *defEval) Eval(ctx Context, formula string) Result {
 	expr := ParseString(formula)
 	if expr != nil {
+		d.addInfo(ctx, expr)
 		return expr.Eval(ctx, d)
 	}
 	return MakeErrorResult(fmt.Sprintf("unable to parse formula %s", formula))
 }
+
+func (d *defEval) addInfo(ctx Context, expr Expression) {
```

Comment:
Please add a short comment why this addInfo is needed. What kind of info is being added and why?
Seems like a pre-requisite for formula evaluation?

Reply:
added

Reply:
comment style

Reply:
fixed